src/platform/platform.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
```

Comment:
Maybe the other platform header could be moved here and this could be renamed to something more gpu-specific?